crates/ruff/src/rules/flake8_bandit/rules/flask_debug_true.rs

```diff
@@ -0,0 +1,89 @@
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::helpers::is_const_true;
+use ruff_python_ast::{Expr, ExprAttribute, ExprCall, Stmt, StmtAssign};
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for uses of `debug=True` in Flask.
+///
+/// ## Why is this bad?
+/// Enabling debug mode shows an interactive debugger in the browser if an error occurs, and allows
```

Comment:
It's annoying, but we manually format these to fit 80 characters, since they sometimes get printed to the terminal.